Library/ViewModels/SignupViewModel.swift

```diff
@@ -4,215 +4,117 @@ import Prelude
 import ReactiveSwift
 import Result
 
-public protocol SignupViewModelInputs {
-  /// Call when the user enters a new email address.
```

Comment:
I got rid of the comments for the sake of the presentation, but the docs can move to document the function's inputs and outputs instead.